spec.html

```diff
@@ -31044,7 +31044,7 @@
             If _obj_ is an exotic object (including Proxy exotic objects) whose behaviour for [[Get]], [[Set]], [[Delete]], and [[GetOwnProperty]] is not the ordinary object implementation of these internal methods.
           </li>
           <li>
-            If any index index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
+            If any index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
```

Comment:
From what I can see both are equally acceptable, in math or otherwise, so that suggests there's no reason to change it.